src/components/radio-group/radio-group.jsx

```diff
@@ -1,13 +1,21 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import injectSheet from 'react-jss';
+import cn from 'classnames';
+
+const styles = {
+  sameRow: {
```

Comment:
prob would had chosen to call it "layout" or "orientation" prop instead with like "horizontal" / "vertical" and default it to vertical

Reply:
Ok I'm on it